keras/src/saving/saving_api.py

```diff
@@ -153,12 +152,16 @@ def load_model(filepath, custom_objects=None, compile=True, safe_mode=True):
     is_keras_zip = str(filepath).endswith(".keras") and zipfile.is_zipfile(
         filepath
     )
+    is_hf_repo = str(filepath).startswith("hf://")
+    if is_keras_zip and is_hf_repo:
```

Comment:
Would this ever be True? As in, would `is_keras_zip` ever resolve to True for a HF path?

Reply:
(closing conversation as snippet is outdated/not relevant anymore)

Reply:
> would is_keras_zip ever resolve to True for a HF path?

It will not, no.